tools/storage/reap_worktrees.sh

```
@@ -0,0 +1,365 @@
+  # had not yet written a file, so the dirty-tree test passed, and its branch had no commits, so the
+  # witness test passed. Its remaining tool calls all failed with "the isolation worktree appears to
+  # have been removed" and the authored fix was lost.
+  #
+  # "Not yet dirty" is not "finished". A worktree belonging to a live workflow run is UNCERTAIN,
+  # and uncertainty means KEEP — the same rule already applied to unparseable locks.
+  wt_base="$(basename "$wt")"
+  case "$wt_base" in
+    wf_*)
+      run_id="${wt_base%-*}"
+      recent="$(find "$HOME/.claude/projects" -type d -name "${run_id}*" -mmin -30 2>/dev/null | head -1)"
+      live_claude="$(ps -Ao comm= 2>/dev/null | grep -c '[c]laude' || true)"
```

Shellcheck (0.11.0)

[info] Consider using pgrep instead of grepping ps output.

(SC2009)